src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2491,6 +2491,223 @@ def oil_water_surface_tension_model(p, t):
             number_of_phases=3 if has_water else 2,
         )
 
+    @staticmethod
+    def create_constant_ph_table(
```

Comment:
How is this to be used?

Reply:
It should be used to create a constant PH table. The funcion `build_constant_pvt_table` in `case_description.py´ will create a constant table depending on the `energy_model_primary_variable` type.

After this PR another one will be created refactoring the PH model. This will probably change this part of the code again.

We discussed that should be better to create an entire `TableDescription` to describre only the `PH tables`.